inc/network_ui/namespace.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace Altis\CMS\Network_UI;
+
+use Altis;
+use WP_Site;
+
+/**
+ * Bootstrap.
+ */
+function bootstrap() {
+	add_filter( 'wpmu_blogs_columns', __NAMESPACE__ . '\\override_columns' );
+	add_filter( 'manage_sites_custom_column', __NAMESPACE__ . '\\handle_column', 10, 2 );
+	add_filter( 'manage_sites_action_links', __NAMESPACE__ . '\\change_row_actions', 0, 2 );
+}
+
+/**
+ * Get configuration for the network UI.
+ *
+ * @return array
+ */
+function get_config() : array {
+	return Altis\get_config()['modules']['cms']['network_ui'];
```

Comment:
We have some inconsistency across module settings - we started out with mostly hyphenated setting names I think. Would be good to double check and do a sweep to standardise these.

Reply:
whups